src/main/notifications/index.ts

```diff
@@ -22,13 +23,23 @@ import MainWindow from '../windows/mainWindow';
 const log = new Logger('Notifications');
 
 class NotificationManager {
-    private mentionsPerChannel: Map<string, Mention> = new Map();
-    private allActiveNotifications: Map<string, Notification> = new Map();
+    private mentionsPerChannel?: Map<string, Mention>;
+    private allActiveNotifications?: Map<string, Notification>;
```

Comment:
Are you nulling these out to ensure that none of this logic runs without having to change every method to check some `this.isEnabled()` before running its logic? If so, I'm not a huge fan of making these nullable and implicitly disabling this logic by deleting them, but I'm 0/5 on that